src/helpers/gamesInfo.js

```diff
@@ -1446,4 +1450,47 @@ export default [
       },
     },
   },
+  {
+    router: {
+      path: "/game/wordshark",
+      component: WordShark,
+      icon: "odnoklassniki",
+    },
+    info: {
+      title: "Word Shark",
+      skills: ["Reading", "Speaking"],
+      dataUsed: ["Vocabulary"],
+      description: "Word Guessing Game",
+      // image: `${baseURL}/v1570941133/TeacherSite/Games/Word_Shark.svg`,
+      image: WordSharkImg,
+      attachments: false,
+      audio: false,
+      completed: true,
+    },
+    keyCuts: [
```

Comment:
I was hoping users could click the letter that was said on their keyboard also. Just need to use the key events hook and dispatch a `Handle_Guess` if it's a letter and it hasn't been guessed already.

Let me know in a comment if that's something you'd want to try including.

Reply:
I'll add it